Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -756,60 +746,60 @@ - (NSString*)actionButtonTitle:(MXKRoomMemberDetailsAction)action
-            title = NSLocalizedStringFromTable(@"room_participants_action_leave", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionLeave];
             break;
         case MXKRoomMemberDetailsActionKick:
             if (self.mxRoom.summary.roomType == MXRoomTypeSpace)
             {
-                title = NSLocalizedStringFromTable(@"space_participants_action_remove", @"Vector", nil);
+                title = [VectorL10n spaceParticipantsActionRemove];
             }
             else
             {
-                title = NSLocalizedStringFromTable(@"room_participants_action_remove", @"Vector", nil);
+                title = [VectorL10n roomParticipantsActionRemove];
             }
             break;
         case MXKRoomMemberDetailsActionBan:
             if (self.mxRoom.summary.roomType == MXRoomTypeSpace)
             {
-                title = NSLocalizedStringFromTable(@"space_participants_action_ban", @"Vector", nil);
+                title = [VectorL10n spaceParticipantsActionBan];
             }
             else
             {
-                title = NSLocalizedStringFromTable(@"room_participants_action_ban", @"Vector", nil);
+                title = [VectorL10n roomParticipantsActionBan];
             }
             break;
         case MXKRoomMemberDetailsActionUnban:
-            title = NSLocalizedStringFromTable(@"room_participants_action_unban", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionUnban];
             break;
         case MXKRoomMemberDetailsActionIgnore:
-            title = NSLocalizedStringFromTable(@"room_participants_action_ignore", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionIgnore];
             break;
         case MXKRoomMemberDetailsActionUnignore:
-            title = NSLocalizedStringFromTable(@"room_participants_action_unignore", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionUnignore];
             break;
         case MXKRoomMemberDetailsActionSetDefaultPowerLevel:
-            title = NSLocalizedStringFromTable(@"room_participants_action_set_default_power_level", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionSetDefaultPowerLevel];
             break;
         case MXKRoomMemberDetailsActionSetModerator:
-            title = NSLocalizedStringFromTable(@"room_participants_action_set_moderator", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionSetModerator];
             break;
         case MXKRoomMemberDetailsActionSetAdmin:
-            title = NSLocalizedStringFromTable(@"room_participants_action_set_admin", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionSetAdmin];
             break;
         case MXKRoomMemberDetailsActionStartChat:
-            title = NSLocalizedStringFromTable(@"room_participants_action_start_chat", @"Vector", nil);
+            title = [VectorL10n roomParticipantsActionStartNewChat];
```

Comment:
`roomParticipantsActionStartChat`?

Reply:
Same as above, couldn't find any "room_participants_action_start_chat", the only thing we have is start new chat.